gluster/cli/peer.py

```diff
@@ -47,6 +47,34 @@ def detach(host):
     cmd = ["detach", host]
     return peer_execute(cmd)
 
+def detachAll():
```

Comment:
Please change the name as detach_all since this convention used in other parts of this lib like bricks.remove_start, bricks.remove_stop etc